src/modules/fancyzones/editor/FancyZonesEditor/MonitorVM.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace FancyZonesEditor
+{
+	public class MonitorVM : INotifyPropertyChanged
+	{
+		public event PropertyChangedEventHandler PropertyChanged;
```

Comment:
this may blow up when null (if not subscribed to).  Add a method that checks for null then call that instead of the event directly.  RaisePropertyChanged("MenuItems"); instead of PropertyChanged();

        public event PropertyChangedEventHandler PropertyChanged;

        private void RaisePropertyChanged(string name)
        {
            PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(name));
        }